api/model/mixins/Dictable.py

```diff
@@ -34,7 +34,4 @@ def as_dict(self, include_pk=True):
 
     @staticmethod
     def _dict_or_collapsed(obj):
-        if obj.dict_collapse:
-            return obj.as_dict(include_pk=False).popitem()[1]
-        else:
-            return obj.as_dict(include_pk=obj.dict_carry_pk)
+        return obj.as_dict(include_pk=False).popitem()[1] if obj.dict_collapse else obj.as_dict(include_pk=obj.dict_carry_pk)
```

Comment:
This was way more readable as multiple lines

Reply:
insignificant change either way